hid-uclogic.c

```diff
@@ -1250,9 +1384,12 @@ static int uclogic_resume(struct hid_device *hdev)
 
 	/* Re-enable tablet, if needed */
 	if (drvdata->tablet_enabled) {
-		__le16 *buf = NULL;
-		rc = uclogic_enable_tablet(hdev, &buf);
-		kfree(buf);
+		if (drvdata->is_hires) {
```

Comment:
The kernel style for blocks of a single line is (regrettably) to omit the braces:

    if (drvdata->is_hires)
            rc = uclogic_enable_tablet(hdev, UCLOGIC_HIRES_PRM_STR_ID, NULL, UCLOGIC_HIRES_PRM_STR_LENGTH);
    else
            rc = uclogic_enable_tablet(hdev, UCLOGIC_PRM_STR_ID, NULL, UCLOGIC_PRM_STR_LENGTH);